ph5/utilities/novitiate.py

```diff
@@ -18,12 +18,15 @@
 from ph5.core import timedoy
 
 
-PROG_VERSION = __version__ = "2019.14"
+PROG_VERSION = __version__ = "2021.84"
 LOGGER = logging.getLogger(__name__)
 try:
-    from PyQt4 import QtGui, QtCore, Qt
+    from PySide2 import QtWidgets, QtCore, Qt
 except Exception:
-    LOGGER.error("PyQt4 must be installed for this to run")
+    msg = ("\n\nNo module named PySide2. "
+           "Please environment_gui.yml to install conda environment"
```

Comment:
"Please use environment_gui.yml to install all required dependencies..." or something similar to that.